bin/create.js

```diff
@@ -219,84 +227,70 @@ function handleCreate (argv) {
                 argv.name = md5.update(fol, 'utf8').digest('hex');
             }
         }
-    }
-    else if (argv.name && argv.output !== 'none') {
-        throw new Error('The `name` option can only be specified when --output is set to `url`.');
+    } else if (argv.name && argv.output !== 'none') {
+        // throw Cli.usageError('The `name` option can only be specified when --output is set to `url`.');
```

Comment:
I am not sure I understand how my suggestion relates to this line of code. What is the intention of this check? Is it `argv.name && argv.output !== 'none'` or `The 'name' option can only be specified when --output is set to 'url'.`